### PR TITLE
Add Authy iOS SDK 1.0.0

### PR DESCRIPTION
**Pull request by AdrianaPineda**

### CODE REVIEW

**Comment by bchen-twilio** [comment]
@AdrianaPineda questions:
- you guys are able to build with iOS 8.0 SDK?
- this PR will be merged once the artifact is uploaded to the production CDN, right?
- one minor comment: could you update the year in the copyright to `2011-2017`?

**Comment by AdrianaPineda** [comment]
Hi @bchen-twilio
- you guys are able to build with iOS 8.0 SDK?
A/ Changed to iOS 8.1
- this PR will be merged once the artifact is uploaded to the production CDN, right?
A/ Correct
- one minor comment: could you update the year in the copyright to 2011-2017?
A/ Changed to 2011-2017

**Comment by bchen-twilio** [comment]
LGTM! 👍

File: TwilioAuthy/1.1.0/TwilioAuthy.podspec [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  s.description  = "The Authy Mobile SDK"
+  s.homepage     = "http://www.twilio.com/"
+  s.platform     = :ios, "8.1"
+  s.license      = { 
+    :type => "Commercial", 
+    :text => "Copyright 2011-2017 Twilio. All rights reserved. Use of this software is subject to the terms and conditions of the Twilio Terms of Service located at http://www.twilio.com/legal/tos"
+  }
+  s.author       = { "Twilio" => "help@twilio.com" }
+  s.source       = { :http    => "https://media.twiliocdn.com/sdk/ios/authy/releases/1.1.0/twilio-authy-ios-1.1.0.tar.bz2" }
+  s.vendored_frameworks   = "AuthyMobileSDK.framework"
```
Comment (ceaglest) [inline TwilioAuthy/1.1.0/TwilioAuthy.podspec]: One thing to note is that the framework name should probably match the name of the Podspec. Right now you have `TwilioAuthy` as the name and `AuthyMobileSDK.framework`.

**Comment by AdrianaPineda** [comment]
@ceaglest thanks! I just made the changes you suggested.

**Comment by AdrianaPineda** [comment]
@bchen-twilio I made some other changes, including the sdk renaming as suggested by @ceaglest. Can you take a look again? Thanks!

File: TwilioAuth/1.0.0/TwilioAuth.podspec [hunk 2]
```diff
@@ -0,0 +1,15 @@
+Pod::Spec.new do |s|
+  s.name         = "TwilioAuth"
+  s.version      = "1.0.0"
+  s.summary      = "Authy Mobile SDK"
+  s.description  = "The Authy Mobile SDK"
+  s.homepage     = "http://www.twilio.com/"
+  s.platform     = :ios, "8.1"
+  s.license      = { 
+    :type => "Commercial", 
+    :text => "Copyright 2011-2017 Twilio. All rights reserved. Use of this software is subject to the terms and conditions of the Twilio Terms of Service located at http://www.twilio.com/legal/tos"
```
Comment (ceaglest) [inline TwilioAuth/1.0.0/TwilioAuth.podspec]: `Copyright 2011-2017 Twilio, Inc`

Comment (AdrianaPineda) [inline TwilioAuth/1.0.0/TwilioAuth.podspec]: @ceaglest thanks, made the change below

**Comment by bchen-twilio** [comment]
LGTM 👍  Great job!

**Pull request merged**